internal/venice-common/src/test/java/com/linkedin/venice/kafka/TopicManagerTest.java

```diff
@@ -127,7 +127,11 @@ private void createTopicManager() {
         .getTopicManager();
   }
 
-  private PubSubProducerAdapter createPubSubProducerAdapter() {
+  protected PubSubProducerAdapter createPubSubProducerAdapter() {
+    Properties props = new Properties();
+    props.put(ApacheKafkaProducerConfig.KAFKA_KEY_SERIALIZER, KafkaKeySerializer.class.getName());
+    props.put(ApacheKafkaProducerConfig.KAFKA_VALUE_SERIALIZER, KafkaValueSerializer.class.getName());
+    props.put(ApacheKafkaProducerConfig.KAFKA_BOOTSTRAP_SERVERS, "localhost:1234");
```

Comment:
Do we need these properties here? they seem unused.

Reply:
Yeah, we should remove this.